_posts/2023-01-03-xtdb-phoenix.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: post
+title: Using XTDB with Phoenix LiveView
+author: tatut
+excerpt: >
+  Immutable Datalog databases are often used in Clojure but other frameworks, like Phoenix LiveView, can benefit from them as well.
+tags:
+ - Elixir
+ - Phoenix LiveView
+ - Datalog
+ - Functional Programming
+---
+
+Relational SQL databases are still the "de facto" choice for most new web applications these days,
+but there are other interesting options to consider. There has been a lot of interest in Clojure circles
+related to Datalog databases, like [Datomic](https://www.datomic.com)
+and [XTDB](https://xtdb.com), which provide a different model of programming especially as it relates
+to time.
+
+Datalog need not only be for Clojure programmers, many programming languages can use it, especially ones
+that have good facilities for describing data.
+In this post, I will give a short introduction to a new library that allows using XTDB with the Erlang
+ecosystem (specifically Elixir) and how to integrate that with the popular [Phoenix Framework](https://phoenixframework.org).
+We will briefly introduce Datalog and the library, then create a sample app with a simple
+LiveView component.
+
+## Datalog, what?
+
+What is datalog then and why should a web developer care about it? Datalog provides a different
```

Comment:
"What is datalog then and why should a web developer care about it?" replicates unneededly the header that says the same thing.